.github/workflows/wiki.yml

```diff
@@ -5,11 +5,30 @@ on:
       - develop
     paths:
       - 'wiki/**'
-  # Triggers this workflow when the wiki is changed
+  # Triggers this workflow when the wiki is changed.
   # (see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#gollum).
   gollum:
 
 jobs:
+  table_of_contents_check:
+    # To verify that the wiki's table of contents matches the headers accurately.
+    name: Check Wiki Table of Contents
+    runs-on: ubuntu-20.04
+    env:
+      CACHE_DIRECTORY: ~/.bazel_cache
```

Comment:
Is the cache being used at all?

Reply:
As discussed, we don't seem to utilize them at the moment, so I've removed them.